numpy/lib/_arraysetops_impl.py

```diff
@@ -222,6 +222,12 @@ def unique(ar, return_index=False, return_inverse=False,
         lexicographical order is chosen - see np.sort for how the lexicographical
         order is defined for complex arrays.
 
+    .. versionchanged: NumPy 2.0
```

Comment:
Remove "numpy" here.

Reply:
Done, also removed it a few lines above from the line I copied this from 😁

Reply:
Ah, yes, that explains it... thanks for fixing that.